src/config/test_config.py

```diff
@@ -4,3 +4,4 @@
 DISABLE_EKS_TESTS = False
 DISABLE_EC2_TESTS = False
 DISABLE_SANITY_TESTS = False
+USE_SCHEDULER = False
```

Comment:
Set this to True so that the results of SageMaker tests on the scheduler can be observed.

Also disable all the other test types apart from SageMaker tests.